### PR TITLE
Translation improved - extended

### PR DESCRIPTION
**Pull request by Knot3n**
German sentences improved, translation expanded.

(Sorry for the inconvenience with the single requets.)

### CODE REVIEW

File: resources/lang/de/base.php [hunk 1]
```diff
@@ -105,12 +105,12 @@
...
                 ],
                 'suspend' => [
-                    'description' => 'Ermöglicht das Suspendieren einer Serverinstanz.',
+                    'description' => 'Ermöglicht das suspendieren einer Serverinstanz.',
                     'title' => 'Server suspendieren',
                 ],
                 'title' => 'Serververwaltung',
                 'unsuspend' => [
-                    'description' => 'Ermöglicht die Suspendierung einer Serverinstanz aufzuheben.',
+                    'description' => 'Ermöglicht die suspendierung einer Serverinstanz aufzuheben.',
```
Comment (Bara) [inline resources/lang/de/base.php]: Why you changed that?

Comment (Knot3n) [reply resources/lang/de/base.php]: Woops, good to see, i think i was reading too fast that line.. 

I'll change it back. 
Thank you for your help

File: resources/lang/de/base.php [hunk 2]
```diff
@@ -106,7 +106,7 @@
                 ],
                 'suspend' => [
                     'description' => 'Ermöglicht das Suspendieren einer Serverinstanz.',
-                    'title' => 'Server suspendieren',
+                    'title' => 'Server Suspendieren',
```
Comment (Bara) [inline resources/lang/de/base.php]: Now is this wrong 😅it's a verb 🤔

Comment (Knot3n) [reply resources/lang/de/base.php]: You confuse me .. :) ... * changed ..

**Comment by DaneEveritt** [comment]
You're missing some empty lines at the top of a few files: https://github.styleci.io/analyses/8w316Z?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Pull request merged**